src/planscape/planning/management/commands/performance_report.py

```diff
@@ -0,0 +1,128 @@
+import os
+from datetime import date
+from django.core.mail import send_mail
+from django.core.management.base import BaseCommand, CommandParser
+from django.db import connection
+from planscape import settings
+from utils.string_utils import snake_to_capitals
+
+
+class Command(BaseCommand):
+    help = "Runs some SQL reports about performance and either emails them or outputs to stdout"
+
+    message_text = ""
+    emailing = False
+    recipient = None
+
+    query_folder = "planning/report_queries/"
```

Comment:
we should add this to gitignore

Reply:
Oh, maybe it's not clear, but this folder actually includes the SQL queries that we use for reporting, so this should be in version control. See the SQL query files in this PR.